src/main/steps/prl-cases/citizenSequence.test.ts

```diff
@@ -11,5 +11,13 @@ describe('applicant1Sequence', () => {
     expect(citizenSequence[1].url).toBe('/dashboard');
     expect(citizenSequence[1].showInSection).toBe('aboutEdgeCase');
     expect(citizenSequence[1].getNextStep({})).toBe('/dashboard');
+
+    expect(citizenSequence[2].url).toBe('/pin-activation/enter-pin');
+    expect(citizenSequence[2].showInSection).toBe('aboutEdgeCase');
+    expect(citizenSequence[2].getNextStep({})).toBe('/pin-activation/case-activated');
+
+    expect(citizenSequence[3].url).toBe('/pin-activation/case-activated');
+    expect(citizenSequence[3].showInSection).toBe('aboutEdgeCase');
+    expect(citizenSequence[3].getNextStep({})).toBe('/dashboard');
```

Comment:
@giorgi-hmcts We need to navigate citizen to respective task-list page based on applicant or respondent. Let's correct it please

Reply:
Fixed, navigation is now done via a custom CaseActivationPostController.